src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XElement.cs

```diff
@@ -1831,7 +1835,7 @@ public override Task WriteToAsync(XmlWriter writer, CancellationToken cancellati
         /// <summary>
         /// This method is obsolete for the IXmlSerializable contract.
         /// </summary>
-        XmlSchema IXmlSerializable.GetSchema()
+        XmlSchema? IXmlSerializable.GetSchema()
```

Comment:
`public static explicit operator int?(XElement element)`

and other similar cast operators above should take nullable `element`